packages/portal-contracts/src/Portal.sol

```diff
@@ -81,6 +84,7 @@ contract Portal is Owned {
         uint256 escrowID,
         uint256 orderID,
         int128 amountSats,
+        int128 amountSatsFilled,
```

Comment:
Wouldn't amountSatsFilled always be zero when an order is first placed?

Reply:
Hmm, this is the OrderMatched event, is that what you're referring to?